export.js

```diff
@@ -59,8 +59,10 @@ module.exports = async ({ file, format, path: p }) => {
   const xml = await readFile(file);
 
   const browser = await puppeteer.launch({
+    executablePath: process.env.CHROMIUM_PATH,
     headless: true,
-  });
+    args: ['--no-sandbox']                                                
```

Comment:
Could you please explain what's the purpose of this arg?

Reply:
Hi, this flag resolves an error when trying to launch chrome headless as root, e.g. see [here](https://github.com/karma-runner/karma-chrome-launcher/issues/158#issuecomment-396914961). This is was I am looking for, too. We want to run draw.io-export within a Docker container but unfortunately can not run it as non-root user (Docker for Windows does not allow writing to mounted volumes from the host). Having the possibility to turn on this flag would be really great. Since we are inside a container and we know which sites are lauched, it should be safe.